test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperTextBoxAreaTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.TestCommon;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
+{
+    public class HtmlHelperTextBoxAreaTest
+    {
+        [Fact]
+        public void TextAreaFor_GeneratesPlaceholderAttribute_WhenDisplayAttributePromptIsSetAndTypeIsValid()
+        {
+            // Arrange            
+            var model = new TextAreaModelWithAPlaceholder();
+            var helper = DefaultTemplatesUtilities.GetHtmlHelper(model);
+
+            // Act
+            var result = HtmlContentUtilities.HtmlContentToString(helper.TextAreaFor(m => m.Property1));
+
+            // Assert 
+            Assert.True(result.Contains(@"placeholder=""HtmlEncode[[placeholder]]"""));
```

Comment:
Instead use `Assert.Contains()` but add `StringComparison.Ordinal`. Also use `Assert.DoesNotContains()` in test below.